src/transformers/tokenization_utils.py

```diff
@@ -675,16 +692,40 @@ def convert_ids_to_tokens(
     def _convert_id_to_token(self, index: int) -> str:
         raise NotImplementedError
 
-    def convert_tokens_to_string(self, tokens: List[str]) -> str:
-        """ Converts a sequence of tokens (string) in a single string.
-            The most simple way to do it is ' '.join(self.convert_ids_to_tokens(token_ids))
-            but we often want to remove sub-word tokenization artifacts at the same time.
+    def convert_tokens_to_string(self, tokens: List[int]) -> str:
```

Comment:
This method add a typehint and a docstring that is not compatible with the way the result is computed (`convert_ids_to_tokens` takes token ids, not tokens). Should the argument `tokens` be renamed `token_ids` or `ids`?

Reply:
I'd go with `token_ids`

Reply:
Actually looking at all the subclasses, it's this implementation that is wrong: all the subclasses get tokens and return strings, so the base class should just join on space and not call `convert_ids_to_tokens`.
@thomwolf or @n1t0, could you confirm?